packages/salesforcedx-vscode-core/package.json

```diff
@@ -170,6 +172,10 @@
           "command": "sfdx.force.function.start",
           "when": "sfdx:project_opened && resource =~ /.*/functions/.*/[^/]+(/[^/]+\\.(ts|js|java|json|toml))?$/ && resourceFilename != package.json && resourceFilename != package-lock.json && resourceFilename != tsconfig.json"
         },
+        {
```

Comment:
Since sfdx.force.function.start is for forceFunctionContainerlessStartCommand, would it make sense to change the command to "sfdx.force.function.containerless.start"?

Reply:
Not sure & just want to raise the pint.  On one have, the name change would be more intuitive to a reader, but on the other hand, "sfdx.force.function.containerless.start" is _**really**_ close to "sfdx.force.function.container.start and might lead to bugs.  Thoughts?

Reply:
I'm fine with making that change.  It would mirror the rest of the code and makes sense 👍